app/src/main/java/org/p2p/wallet/restore/ui/keys/adapter/SeedPhraseWatcher.kt

```diff
@@ -15,16 +15,18 @@ import kotlin.properties.Delegates
 
 class SeedPhraseWatcher(
     private val onKeyAdded: (SecretKey) -> Unit,
-    private val onSeedPhraseInserted: (List<SecretKey>) -> Unit
+    private val onSeedPhraseInserted: (List<SecretKey>) -> Unit,
+    var isLast: Boolean
 ) : TextWatcher {
 
     companion object {
         fun installOn(
+            isLast: Boolean,
```

Comment:
can't get idea what should be last here.
isLastWord or isLastPhrase?

Reply:
Yep, `isLastKey` would be representative